db/migrate/20151102113518_add_default_value_to_minimum_fixed_fee.rb

```diff
@@ -0,0 +1,12 @@
+class AddDefaultValueToMinimumFixedFee < ActiveRecord::Migration
+  def up
+    firms = Firm.where(minimum_fixed_fee: nil)
```

Comment:
Will that need an `attr_accessible :minimum_fixed_fee` incase the column doesn't exist?